packages/react-liveness/package.json

```diff
@@ -43,7 +43,7 @@
     "typecheck": "tsc --noEmit"
   },
   "peerDependencies": {
-    "aws-amplify": "^5.0.1",
+    "aws-amplify": "console-preview",
```

Comment:
Can we keep this pinned to v5? We want to test and see if we can keep liveness on v5

Reply:
don't think we can have 2 versions of `aws-amplify` without conflicts...

Reply:
@thaddmt Can you try this in a follow up pr?